package.json

```diff
@@ -20,18 +20,23 @@
   "dependencies": {
     "aes-js": "^3.1.2",
     "bfx-api-node-core": "^1.5.1",
+    "bfx-api-node-plugin-wd": "^1.0.4",
     "bfx-api-node-rest": "^4.1.2",
     "bfx-hf-algo": "git+https://github.com/bitfinexcom/bfx-hf-algo.git#v4.6.0",
     "bfx-hf-ext-plugin-dummy": "github:bitfinexcom/bfx-hf-ext-plugin-dummy#v1.0.4",
+    "bfx-hf-indicators": "git+https://github.com/bitfinexcom/bfx-hf-indicators.git#v2.0.9",
     "bfx-hf-models": "git+https://github.com/bitfinexcom/bfx-hf-models.git#v2.4.1",
     "bfx-hf-models-adapter-lowdb": "git+https://github.com/bitfinexcom/bfx-hf-models-adapter-lowdb.git#v1.0.5",
+    "bfx-hf-strategy": "git+https://github.com/avsek477/bfx-hf-strategy.git#util-function-update",
```

Comment:
need to update the package while after merge: (https://github.com/bitfinexcom/bfx-hf-strategy/pull/20)

Reply:
sure

Reply:
done @vigan-abd